config/middlewares/authorization.js

```diff
@@ -1,33 +1,46 @@
+import jwt from 'jsonwebtoken';
+import config from '../env/all';
+
+const { secret } = config;
+
 /**
  * Generic require login routing middleware
+ * @export
+ *
+ * @param {any} req request sent from the body
```

Comment:
when writing JSDOC is great practice to include the appropriate data type and not `any`